examples/RESOURCES.md

```diff
@@ -0,0 +1,11 @@
+# Resources and examples
+
+> Useful tools or projects as example of main Puppeteer features.
+
+## Web scraping
+- [Puppetron](https://github.com/cheeaun/puppetron) - Non-production (Headless Chrome Node API)-based rendering solution.
```

Comment:
Demo site that shows how to use Puppeteer and Headless Chrome to render pages. Inspired by https://github.com/GoogleChrome/rendertron.